src/components/subtitleuploader/subtitleuploader.js

```diff
@@ -75,7 +77,20 @@ function setFiles(page, files) {
     reader.readAsDataURL(file);
 }
 
-function onSubmit(e) {
+function getStringFromFile(file) {
```

Comment:
It would probably make sense to move this function to a dedicated file in the `utils` directory... probably `/src/utils/file.ts`